core/src/main/java/ai/timefold/solver/core/api/score/stream/common/IntervalBreak.java

```diff
@@ -11,21 +11,21 @@ public interface IntervalBreak<Interval_, Point_ extends Comparable<Point_>, Dif
     /**
      * @return never null, the interval cluster leading directly into this
      */
-    IntervalCluster<Interval_, Point_, Difference_> getPreviousIntervalCluster();
+    ConcurrentUsage<Interval_, Point_, Difference_> getPreviousConcurrentUsage();
```

Comment:
getPreviousRange?

Reply:
Huh, I thought I deleted this file; it should be unused (in particular, the collector uses the same break interface as consecutive)